cmd/update-index/main.go

```diff
@@ -102,12 +102,12 @@ func (b Binary) Row() string {
 	<td>%s</td>
 	<td>%s</td>
 	<td>%s</td>
-	<td>%s</td>
-	<td><a class="copy" href="%s">copy link</a></td>`, b.Version, b.OperatingSystem, b.Architecture, b.Name, b.downloadLink())
+    <td><a href="https://%s">%s</a></td>
+    <td><i class="fa">&#xf0c5;</i><a class="copy" href="https://%s">  %s</a></td>`, b.Version, b.OperatingSystem, b.Architecture, b.downloadLink(), b.Name, b.downloadLink(), b.downloadLink())
```

Comment:
Since this project uses bulma.io, this can be more readable by using:

```
<td><span class="icon"><i class="far fa-copy"></i></span>...
```

Reply:
fixed up